tests/tests/fv3_ccpp_regional_control

```diff
@@ -24,6 +24,10 @@ export OZ_PHYS_OLD=.F.
 export OZ_PHYS_NEW=.T.
 export H2O_PHYS=.T.
 export HYBEDMF=.F.
+export RESTART_INTERVAL=12
+export POSTAPP='lam'
+export QUILTING=.false.
```

Comment:
If `QUILTING` is false, that means write component is not used, which means no inline post. Why is then `POSTAPP` set on the previous line?